### PR TITLE
AAE-10134: turn dependabot and version propagation GH reusable workfl…

### PR DESCRIPTION
**Pull request by atchertchian**
…ows into actions

### CODE REVIEW

**Review by gionn — APPROVED** [review]

**Review by erdemedeiros — APPROVED** [review]
LGTM!

**Comment by atchertchian** [comment]
I'll rebase that PR and test it on a repository before merging (and updating multiple repos to use the new actions)

File: .github/actions/automate-propagation/action.yml [hunk 1]
```diff
@@ -22,12 +22,11 @@ runs:
     - name: Check is propagation PR
       id: check
       env:
-        EVENT_NAME: ${{ github.event_name }}
         LABEL: ${{ github.event.label.name }}
         PR_LOGIN: ${{ github.event.pull_request.user.login }}
       shell: bash
       run: |
-        if [[ $EVENT_NAME == 'labeled' && $LABEL == 'updatebot' && $PR_LOGIN == 'alfresco-build' ]]
+        if [[ $LABEL == 'updatebot' && $PR_LOGIN == 'alfresco-build' ]]
```
Comment (atchertchian) [inline .github/actions/automate-propagation/action.yml:29]: @erdemedeiros wdyt of this change please?
When testing on a PR I noticed the event name was "pull_request" and this check was not on the original workfow (the workflow trigger was on this event type though)

**Comment by atchertchian** [comment]
TODO: update README

File: .github/actions/automate-propagation/action.yml (hunk 1 above)
Comment (erdemedeiros) [reply .github/actions/automate-propagation/action.yml:29]: @atchertchian I don't fully understand your comment. The check was done globally [at the job level](https://github.com/Alfresco/alfresco-build-tools/pull/104/files#diff-407b2f2e7c243f844d95ba084196232884957fe2b3c90a9ac926efb25cfce721L18). Unfortunately it doesn't seem possible to have a global `if` at the action level.

Comment (atchertchian) [reply .github/actions/automate-propagation/action.yml:29]: I had initially added this in the action, I will put it back globally on the caller workflow so we should be good.

**Comment by atchertchian** [comment]
Keeping the work branch for now as there are currently running workflows that are still running on it

**Pull request merged**